### DISCUSSION TITLE
Click those squares?! (Graphical glitches on Pixel 7a)

### DISCUSSION THREAD

**Discussion by Ekischleki** [ppy/osu#26966] [Q&A] [answered]
I have a google pixel 7a and am trying to run osu lazer on it. Here's a screen recording of what it looks like:
(WARNING, FLASHING SCREEN!)
https://photos.app.goo.gl/DfjAU18A5KaYTftm7

I don't want to create an issue, as it is probably hardware/os related. Still, anything I can do about this?
Here are all the logs that were in the log directory of my android phone
[osu logs.zip](https://github.com/ppy/osu/files/14142744/osu.logs.zip)
About my phone (from about this phone settings):
Device name: Pixel 7a
Hardware version: MP1.0
Manufactured year: 2023
About my os
Android 14
Android Security update Jan., 5, 2024
Google play system update Jan., 1, 2024

I hope this is all you need, otherwise write a comment ^-^

Thank you

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/25189

**Reply by Ekischleki** [reply]
Whoops, sorry! Didn't see that post hehe
